File: ObjectProgrammingJava/projekt po - 2/src/projekt/zwierzeta/Wilk.java
package projekt.zwierzeta;

import projekt.Organizm;
import projekt.Wypisanie;
import projekt.swiat.Swiat;

import java.awt.*;

public class Wilk extends Zwierze implements Wypisanie {

    public Wilk(Swiat swiat, int x, int y, boolean czyZyje){
        polozenieX = x;
        polozenieY = y;
        sila = 9;
        inicjatywa = 5;
        kolor = Color.black;
        oznaczenie = 'W';
        this.swiat = swiat;
        this.czyZyje = czyZyje;
        this.czyMozeRozmnazac = 0;
        wiek = 0;
    }

    @Override
    public String Wypisz() {
        return "Wilk" + getKoncowka();
    }
}

File: ObjectProgrammingPython/Zwierzeta/wilk.py
from Zwierzeta.zwierze import Zwierze
from organizm import Organizm

class Wilk(Zwierze):
    def __init__(self, Swiat, x, y, czyZyje):
        super(Wilk, self).__init__(Swiat, x, y, czyZyje)
        self.sila = 9
        self.inicjatywa = 5
        self.kolor = "black"
        self.oznaczenie = "W"
        self.czyMozeRozmnazac = 0
        self.wiek = 0

    def Wypisz(self):
        return "Wilk" + self.koncowka()



        